docs/how-to/load-models-from-pvc.md

```diff
@@ -0,0 +1,61 @@
+# Load Models from PVC
```

Comment:
Lets make this document less abstract and more concrete.

Example: `if you ran `ollama pull qwen:0.5b` to preload` --> Too abstract, where does this cmd store the models? how does that interoperate with their k8s spec for their loader job? Better yet, avoid narrating the process in english and just point the user at a specific example of how to do this (we need a ollama pvc loader example here: https://github.com/substratusai/kubeai/tree/main/examples).

Example: `Use a storage class that supports your access mode requirements` --> better yet, just tell them what it should be - we know that KubeAI has a read-many requirement in 90% of cases.

Reply:
done